apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -85,6 +85,47 @@ type TempoStackSpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Tenants Configuration"
 	Tenants *TenantsSpec `json:"tenants,omitempty"`
+
+	// ObservabilitySpec defines how telemetry data gets handled.
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Observability"
+	Observability ObservabilitySpec `json:"observability,omitempty"`
+}
+
+// ObservabilitySpec defines how telemetry data gets handled.
+type ObservabilitySpec struct {
+	// Operands defines how operand telemetry data gets handled.
+	Operands ObservabilityOperandsSpec `json:"operands,omitempty"`
```

Comment:
Why is the operand nesting needed? The CR is implicitly a config for operands only.

Reply:
true :X